src/components/bulk-email-tool/bulk-email-form/bulk-email-recipient/components/EmailList/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import {
+  Form, Chip, Container, Button,
+} from '@edx/paragon';
+import { Person, Close, SupervisedUserCircle } from '@edx/paragon/icons';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { v4 as uniqueId } from 'uuid';
+import messages from '../../../messages';
+
+import { isValidEmail } from './utils';
+import './styles.scss';
+
+function EmailList(props) {
+  const {
+    handleEmailSelected, emailList, handleDeleteEmail,
+    intl,
+  } = props;
+
+  const [emailListAdded, setEmailListAdded] = useState(false);
+  const [emailInputValue, setEmailInputValue] = useState('');
+  const [invalidEmailError, setInvalidEmailError] = useState(false);
+
+  const handleDeleteEmailSelected = (id) => {
+    if (handleDeleteEmail) {
+      handleDeleteEmail(id);
+    }
+  };
+
+  const handleChangeEmailInput = ({ target: { value } }) => {
+    setEmailInputValue(value);
+    if (isValidEmail(value)) {
+      setInvalidEmailError(false);
+    }
+  };
+
+  const handleAddEmail = () => {
+    if (emailInputValue.length) {
+      if (!emailListAdded) {
+        setEmailListAdded(true);
+      }
+      if (isValidEmail(emailInputValue)) {
+        const emailFormatted = emailInputValue.toLocaleLowerCase();
+        const data = { id: uniqueId(), email: emailFormatted };
+        handleEmailSelected(data);
+        setInvalidEmailError(false);
+      } else {
+        setInvalidEmailError(true);
+      }
+    }
+  };
```

Comment:
```suggestion
  const handleAddEmail = () => {
    if (!emailInputValue.length) return;
      if (!emailListAdded) {
        setEmailListAdded(true);
      }
      if (isValidEmail(emailInputValue)) {
        const emailFormatted = emailInputValue.toLocaleLowerCase();
        const data = { id: uniqueId(), email: emailFormatted };
        handleEmailSelected(data);
        setInvalidEmailError(false);
      } else {
        setInvalidEmailError(true);
      }
  };
```
This way we don't have nested ifs that are more difficult to read